### PR TITLE
ISO Mdl presentation adjustments to acommodate changes in Rust API

### PR DESCRIPTION
**Pull request by rschulman**

### CODE REVIEW

**Comment by rschulman** [comment]
This has been tested against the Idemia and GET readers and is working.

File: project.yml [hunk 1]
```diff
@@ -3,10 +3,10 @@ options:
   bundleIdPrefix: com.spruceid.mobile
 packages:
   SpruceIDMobileSdk:
-    url: https://github.com/spruceid/mobile-sdk-swift
+    #url: https://github.com/spruceid/mobile-sdk-swift
     # from: 0.0.10
-    revision: 379b4a3d4550df6ab9da0e5be3406d8021dcd253
-    # path: "../mobile-sdk-swift"
+    #revision: 379b4a3d4550df6ab9da0e5be3406d8021dcd253
+    path: "../mobile-sdk-swift"
```
Comment (Juliano1612) [inline project.yml:9]: Can we point to a remote version? If we merge with a fixed local version, it will cause some problems for others

**Review by Juliano1612 — CHANGES_REQUESTED** [review]

File: project.yml (hunk 1 above)
Comment (rschulman) [reply project.yml]: Good catch, thanks! Fixed.

File: project.yml [hunk 2]
```diff
@@ -4,9 +4,9 @@ options:
 packages:
   SpruceIDMobileSdk:
     url: https://github.com/spruceid/mobile-sdk-swift
-    # from: 0.0.10
-    revision: 379b4a3d4550df6ab9da0e5be3406d8021dcd253
-    # path: "../mobile-sdk-swift"
+     from: 0.0.10
```
Comment (Juliano1612) [inline project.yml:7]:
```suggestion
    from: 0.0.11
```

**Review by Juliano1612 — CHANGES_REQUESTED** [review]

**Review by Juliano1612 — APPROVED** [review]

**Pull request merged**